reference/Upload-API.yaml

```diff
@@ -1448,5 +1433,138 @@ paths:
               type: object
               properties: {}
       description: 'Returns the list of entitlement and Time to live (TTL) for the "entry" identify by the entryid. '
+  '/api/v1/{apiUserId}/{apiKey}/{languageCode}/feed/{feedId}/':
+    parameters:
+      - schema:
+          type: string
+        name: apiUserId
+        in: path
+        required: true
+      - schema:
+          type: string
+        name: apiKey
+        in: path
+        required: true
+      - schema:
+          type: string
+        name: languageCode
+        in: path
+        required: true
+      - schema:
+          type: string
+        name: feedId
+        in: path
+        required: true
+    get:
+      summary: Unrestricted Feed
+      tags:
+        - Feed
+      responses:
+        '200':
```

Comment:
Does this API have any other responses other than success (200)? If yes, can we document them as well please?